packages/js/manifests/polywrap/src/formats/polywrap/migrators/0.3.0_to_0.4.0.ts

```diff
@@ -0,0 +1,15 @@
+import { ILogger } from "@polywrap/logging-js";
+import { PolywrapManifest as OldManifest } from "../0.3.0";
+import { PolywrapManifest as NewManifest } from "../0.4.0";
+
+export function migrate(migrate: OldManifest, logger?: ILogger): NewManifest {
+  if (migrate.extensions?.deploy) {
+    logger?.warn(
+      `The Polywrap Deploy Manifest does not needs to be defined in the Polywrap Manifest, please remove it`
```

Comment:
I think it would make more sense to just
```typescript
delete migrate.extensions.deploy;
```
instead of warning the user? Or you can do both!

If you still want to keep the warning text, there's an English grammar error in your sentence:
`needs` -> `need`

An alternative sentence too:
```
The Polywrap Deploy Manifest ('extensions.deploy') is no longer used within the Polywrap Manifest. Please remove it from your manifest file.
```

Reply:
just pushed changes based on your feedback, please feel free to review and resolve if you feel it's enough :smile: